sdk/containers/azcontainerregistry/README.md

```diff
@@ -38,11 +38,11 @@ az acr create --name MyContainerRegistry --resource-group MyResourceGroup --loca
 ```
 ### Authentication
 
-This document demonstrates using [azidentity.NewDefaultAzureCredential](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#NewDefaultAzureCredential) to authenticate. 
-This credential type works in both local development and production environments. 
+This document demonstrates using [azidentity.NewDefaultAzureCredential](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#NewDefaultAzureCredential) to authenticate.
+This credential type works in both local development and production environments.
 We recommend using a [managed identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) in production.
 
-Client and BlobClient accepts any [azidentity][https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity] credential. 
+[Client](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/containers/azcontainerregistry#Client) and [BlobClient](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/containers/azcontainerregistry#BlobClient) accepts any [azidentity][https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity] credential.
 See the [azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity) documentation for more information about other credential types.
```

Comment:
can we paste env var that needed to make the default credential work quickly?

Reply:
I don't prefer to do so. It is related with auth process that should be documented in `azidentity`.

Reply:
:+1: it's best to use `DefaultAzureCredential` as simply as possible in client library docs. We don't want to imply that environment variables are the only or the best way to configure authentication, or to describe the other options here.

Reply:
Since this is the overall policy across data plane, I'm ok we have a link to azidentity